esp/services/ws_esdlconfig/ws_esdlconfigservice.hpp

```diff
@@ -57,17 +57,26 @@ class CWsESDLConfigEx : public CWsESDLConfig
 
     bool attachServiceToDali() override
```

Comment:
Would it make sense to create a CDaliService class, that encapsulates the flag, the synchronization and the accessor methods?

Reply:
that's a good point, we do anticipate more methods making use of that logic, can you open a jira to propose that improvement? Thanks.